packages/ui/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte

```diff
@@ -121,6 +122,13 @@
 
 	$: hasHadSelection = hasHadSelection || $selectedOptions.length > 0;
 	$: if ($selectedOptions && hasHadSelection) {
+		// console.log(label, opts.length, $selectedOptions);
+		// console.log($options);
+		// values should be $selectedOptions that are currently available in the opts array
+
+		if ($selectedOptions.length > opts.length) {
+			updateSelectedOptions(opts);
```

Comment:
I'm a little concerned that i'm passing in opts here, which contains the updated query array. Just feels weird to me

Reply:
@ItsMeBrianD do you have any thoughts here? Doesn't seem bad to me, but there may be a cleaner way

Reply:
@ItsMeBrianD Do you have any thoughts here? I feel like there may be a cleaner implementation, but I can't put my finger on it